Cython/Utility/FunctionArguments.c

```diff
@@ -422,7 +422,7 @@ static int __Pyx_MergeKeywords(PyObject *kwdict, PyObject *source_mapping) {
 #if CYTHON_METH_FASTCALL
     #define __Pyx_Arg_FASTCALL(args, i) args[i]
     #define __Pyx_NumKwargs_FASTCALL(kwds) PyTuple_GET_SIZE(kwds)
-    #define __Pyx_KwValues_FASTCALL(args, nargs) (&args[nargs])
+    #define __Pyx_KwValues_FASTCALL(args, nargs) (args+nargs)
```

Comment:
I know, the macro is never called on anything but plain variables, but it's cheap to add parentheses anyway and makes it safer to change the usage.
```suggestion
    #define __Pyx_KwValues_FASTCALL(args, nargs) ((args) + (nargs))
```